tests/bluetooth/tester/src/btp/btp_ias.h

```diff
@@ -10,7 +10,7 @@
 #include <stdint.h>
 
 /* events */
-#define IAS_EV_OUT_ALERT_ACTION       0x80
-struct ias_alert_action_ev {
+#define BTP_IAS_EV_OUT_ALERT_ACTION	0x80
```

Comment:
General comment: In Zephyr we use (or should use) spaces rather than tabs to indent after a non-tab character